src/data/metadata.csv

```diff
@@ -2295,14 +2295,53 @@ LT,LT,Lithuania,,,,,,-1
 LU,LU,Luxembourg,,,,,,-1
 LV,LV,Latvia,,,,,,-1
 LY,LY,Libya,,,,,,-1
+LY_AB,LY,Libya,AB,Abyar,,,Abyar,
+LY_AJ,LY,Libya,AJ,Ajdabiya,,,Ajdabiya,
+LY_AM,LY,Libya,AM,Abyar,,,Al Abyar,
 LY_BA,LY,Libya,BA,Benghazi,,,Benghazi,
-LY_JI,LY,Libya,JI,Jafara,,,Al-Jafara,
-LY_MS,LY,Libya,MS,Misrata District,,,Misrata,
+LY_BJ,LY,Libya,BJ,Bin Jawad,,,Bin Jawad,
+LY_BR,LY,Libya,BR,Brega,,,Brega,
+LY_BU,LY,Libya,BU,Butnan District,,,Butnan,
+LY_BW,LY,Libya,BW,Bani Walid,,,Bani Walid,
+LY_BY,LY,Libya,BY,Bayda District,,,Al Bayda,
+LY_DR,LY,Libya,DR,Derna District,,,Derna,
+LY_GD,LY,Libya,GD,Ghadames,,,Ghadames,
+LY_GH,LY,Libya,GH,Gharyan,,,Gharyan,
+LY_GT,LY,Libya,GT,Ghat District,,,Ghat,
+LY_HA,LY,Libya,HA,Harawa,,,Harawa,
+LY_JA,LY,Libya,JA,Jabal al Akhdar,,,Jabal al Akhdar,
+LY_JG,LY,Libya,JG,Jabal al Gharbi,,,Jabal al Gharbi,
+LY_JI,LY,Libya,JI,Jafara,,,Al-Jafra,
+LY_JN,LY,Libya,JN,Janzur,,,Janzur,
+LY_JU,LY,Libya,JU,Jufra,,,Al Jufrah,
+LY_KA,LY,Libya,KA,Kabaw District,,,Kabaw,
```

Comment:
Can you make sure hierarchical order is preserved? For example, it looks like Kabaw is a town, not a district, and it should be part of Nalut so the record should be:
```
LY_NL_KA,LY,Libya,NL,Nalut District,KA,Kabaw,,
```

https://en.wikipedia.org/wiki/Kabaw

These appear to be the districts in Libya: https://en.wikipedia.org/wiki/Districts_of_Libya

Reply:
Fixed the order and I've not made subregion2 as the name to match against in the test. However looks like I'm not able to run any test?

http://screen/AaazBN21MsM

Reply:
You should be using Python 3 instead :-)

Reply:
See here for instructions to setup your local dev environment, although I think the part about installing dependencies needs to be updated: https://github.com/open-covid-19/data/tree/main/src#development-environment